app/models/zip_checksum.rb

```diff
@@ -0,0 +1,10 @@
+##
+# ZipChecksum contains the MD5 checksum and creation info for every preserved copy (druid, version)
+# that has been archived.
+class ZipChecksum < ApplicationRecord
+  belongs_to :preserved_copy
+
+  validates :preserved_copy, presence: true
+  validates :md5, presence: true
```

Comment:
we know more about md5 that we can validate for.   We could use:
- length validator
- format validator to ensure it is always a well-formed MD5 (more expensive to check than length, but not too bad?)
- uniqueness ?
- strict?

Reply:
Strict Validation: You can also specify validations to be strict and raise ActiveModel::StrictValidationFailed when the object is invalid.

Do we want this?

Reply:
uh - how do we report now?  how would our reporting change w strict?

Reply:
For now, I just added the format validator.